### PR TITLE
Add Custom Songs page

### PR DESCRIPTION
**Pull request by TheCzar1994**
Adds updated dedicated Custom Songs page/URL as was noted critical by Helen here: https://discord.com/channels/452928402203344908/752003683780395069/1200901546406334564

Also updates MOTW again 😄

### CODE REVIEW

File: src/collections/map-of-the-week/mon-mar-18-2024-13-00-00-gmt-0600-central-standard-time.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+mapId: 3a809
+review: The top diff has a lot of energy, variety in patterns, and absolutely represents the spirit of Balanced maps with moderately fast patterns, some body movement, and engaging patterns. The two downmaps provide a nice progression that will still prove challenging and intense to less experienced players.
+startDate: 2024-03-11T19:00:00.000Z
```
Comment (TimothyGillespie) [inline src/collections/map-of-the-week/mon-mar-18-2024-13-00-00-gmt-0600-central-standard-time.md:4]: The date seems wrong. Shouldn't it be 2024-03-18?

File: src/collections/posts/custom-songs.md [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+## Custom Songs for Quest VR Users
+Getting custom songs on the Quest requires a little more work than with its PC cousins. There are two pieces of software you’ll need: [SideQuest](https://sidequestvr.com/setup-howto) and [BMBF](https://bmbf.dev/stable) or [QuestPatcher](https://github.com/Lauriethefish/QuestPatcher/releases).
+
+> **💡 HINT: When downloading SideQuest, be sure to use the "Advanced Installer". This will greatly help you when adding songs!**
+
+Once you have the right software installed, you have a couple of options on how to access songs. The [Beat Saber Modding Group Wiki](https://bsmg.wiki/quest-modding.html#installing-songs) is the best place to get the latest information, but here’s a quick step-by-step:
+
+1. Make sure you have the latest BeatSaber mods available for your moddable game. Keep in mind that Quest modding is usually well behind PC modding so you will likely need to downgrade. You can find the latest mods by clicking [here](https://computerelite.github.io/tools/Beat_Saber/questmods.html)
+
+2. Using QuestPatcher, click Browse and open the mod pack you downloaded from the link above
```
Comment (TimothyGillespie) [inline src/collections/posts/custom-songs.md:135]: It seems a bit inconsistent. Here you don't quote "Browser" but you quote "MORE SONGS" under 3.

File: src/collections/posts/custom-songs.md [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+
+3. Now, from within the game, you can click on "MORE SONGS" to search and download songs
+
+You can also search for songs using the method listed above at **"Custom Songs with the Base Game (More difficult!)** and then manually load them into your headset using SideQuest and finding your custom songs folder there, however, we don't typically recommend this option.
+
+If you find videos more helpful, check out this video by XORoseGold on How To Get Custom Songs in BeatSaber, uploaded on March 7, 2024!
```
Comment (TimothyGillespie) [inline src/collections/posts/custom-songs.md:141]: I think the first O should be lower case.

**Review by TimothyGillespie — CHANGES_REQUESTED** [review]

**Comment by TimothyGillespie** [comment]
@Pixelguymm the code in `app.scss` and `PostPage.svelte` are mine so that'd be a self-review on these.

File: src/collections/map-of-the-week/mon-mar-18-2024-13-00-00-gmt-0600-central-standard-time.md (hunk 1 above)
Comment (TheCzar1994) [reply src/collections/map-of-the-week/mon-mar-18-2024-13-00-00-gmt-0600-central-standard-time.md:4]: Good catch! Yes! Will fix

**Review by Pixelguymm — APPROVED** [review]

**Pull request merged**